ops/ansible/vars.yml

```diff
@@ -21,6 +21,7 @@ migrate_target: "db-migrate-prod"
 seed_txt_clean_target: "txt-clean"
 seed_txt_clone_target: "txt-clone"
 seed_txt_apply_target: "db-txt-prod"
+seed_txt_trunc_target: "db-txt-trunc-prod"
```

Comment:
I probably should have talked through this in more detail. I think we (probably?) want to make the trunc target optional, for two reasons:

1. Since the `db-txt-prod` target doesn't do anything if the TXT files (translations, in particular) haven't changed. But the trunc target will blindly destroy all the looping templates, every time. Over the course of enough deployments, this will lead to garbage in the database, since deleted records are not deleted from disk in XTDB. 
2. It's also a slightly risky action to take. If, for whatever reason, the `Daily_emails_RSS` repo were to become corrupt, deleting all the looping templates would mean we need to restore the `Daily_emails_RSS` repo before they could be replaced. I think this situation is very unlikely, but it still implies we should be in complete (manual) control of the truncation process, all the same.

What do you think, @balwa ?

Reply:
@deobald yes, this is exactly how it is added at this point. By default the action shall not run. It will run only when the person deploying manually requests the resources to be recreated, using taints. I have added this as a comment above the resource definition in terraform.

Reply:
https://github.com/pariyatti/kosa/pull/85/files#diff-cbe97095f2b1444a38119f03e54f599e6b05499f52286f1d900b7c44c6bca327R55-R60

Reply:
Sorry! That makes sense. :)